js/ui/file_manager/file_items_controller.js

```diff
@@ -281,14 +312,14 @@ export default class FileItemsController {
     }
 
     _createDirectoryInfo(fileItem, parentDirectoryInfo) {
-        return extend(this._createFileInfo(fileItem, parentDirectoryInfo), {
+        return extend(this.createFileInfo(fileItem, parentDirectoryInfo), {
             icon: "folder",
             expanded: fileItem.isRoot,
             items: [ ]
         });
     }
 
-    _createFileInfo(fileItem, parentDirectoryInfo) {
+    createFileInfo(fileItem, parentDirectoryInfo) {
```

Comment:
The `createFileInfo` method should be private. The main idea, that only controller object should get info about file items. Also, the uploading mechanism goes throw file provider. Therefore, I couldn't see any reasons to make the method as public.

Reply:
Fixed